### PR TITLE
kubecost cloud update

### PR DESCRIPTION
**Pull request by jessegoodier**

### CODE REVIEW

**Review by srpomeroy — APPROVED** [review]

File: kubecost-cloud-architecture.md [hunk 1]
```diff
@@ -1,21 +1,17 @@
 Kubecost Cloud Architecture Overview
 ===================================
 
-Below are the major components deployed with the [Kubecost Helm chart](http://docs.kubecost.com/install) for Kubecost Cloud, excluding certain Enterprise components such as durable storage:
+Kubecost Cloud uses an agent to gather metrics and send them to an in-region s3 bucket.
```
Comment (srpomeroy) [inline kubecost-cloud-architecture.md:4]: Do we need to specify in-region?

**Review by srpomeroy — APPROVED** [review]
One optional nit.

File: kubecost-cloud-architecture.md (hunk 1 above)
Comment (bstuder99) [reply kubecost-cloud-architecture.md:4]: *S3 bucket

File: kubecost-cloud-architecture.md [hunk 2]
```diff
@@ -1,21 +1,17 @@
...
 
-1. **Kubecost Agent Pod Pod**  
-    a. Cost-model: provides cost allocation calculations and metrics, both reads and writes to Prometheus
-2. **Prometheus**  
-    a. Prometheus server: time-series data store for cost and health metrics
```
Comment (bstuder99) [inline kubecost-cloud-architecture.md:9]: daemonSet

**Pull request merged**